src/mixins/filing-template-mixin.ts

```diff
@@ -689,7 +695,8 @@ export default class FilingTemplateMixin extends DateMixin {
         name: FilingTypes.CONVERSION,
         certifiedBy: this.getCertifyState.certifiedBy,
         date: this.getCurrentDate, // "absolute day" (YYYY-MM-DD in Pacific time)
-        folioNumber: '' // not applicable to SP/GP
+        folioNumber: '', // not applicable to SP/GP
+        documentIdState: this.getDocumentIdState // document id will serve as a barcode number
```

Comment:
This (and the other above) is for the "build" (save) functions. Do you need to add something to the "parse" (load) functions for resuming a draft?

Reply:
Wait. I could 6 places where you save document id state but only 5 places where you restore it.